app/javascript/src/components/dashboard/Hours.jsx

```diff
@@ -0,0 +1,242 @@
+  trophy(user, view) {
+    let rank;
+    if (this.state.total) {
+      rank = view == "hour"
+        ? user.rank.thour
+        : user.rank.tnight;
+    } else {
+      rank = view == "hour"
+        ? user.rank.whour
+        : user.rank.wnight;
+    }
+    if (rank == 1) {
+      return <Icon name='trophy' color='yellow'/>;
+    } else if (rank == 2) {
+      return <Icon name='trophy' color='grey'/>;
+    } else if (rank == 3) {
+      return <Icon name='trophy' color='brown'/>;
+    }
+  };
+
+  //Function sorting Data from Greatest to Least by the given Rank and Specific View (Week or Total)
+  handleNightorHours = (e, timeOfDay) => {
+    let newNight; //new instance of order list used to alter State Hours value
+    if (this.state.total) { //Order by Total Hours or Night
+      newNight = getHours().sort((b, a) => {
+        const timeA = timeOfDay === 'Night'
+          ? a.night
+          : a.hours;
+        const timeB = timeOfDay === 'Night'
+          ? b.night
+          : b.hours;
+
+        return timeA.total - timeB.total;
+      });
+    } else { //Order by Total Hours or Night
+      newNight = getHours().sort((a, b) => {
+        const timeA = timeOfDay === 'Night'
+          ? a.night
+          : a.hours;
+        const timeB = timeOfDay === 'Night'
+          ? b.night
+          : b.hours;
+        return timeB.week - timeA.week;
+      });
+    }
+    this.setState({rank: timeOfDay, hours: newNight});
+  };
+
+  //Each User Hours and Night count displayed for a given view customCard = (props) => {
+  customCard = (props) => {
+  return <Card key={props.user.name} style={{
+        width: "180px"
+      }}>
+      <Card.Content >
+        <Card.Header>{props.user.name}
+        </Card.Header>
+      </Card.Content>
+      <Card.Content>
+        {this.state.total == false ? "Week " : "Total "}
+        <Card.Description>
+          {this.hourView()}
+          <strong>{this.hoursNumber(props.user)}
+          </strong>
+        </Card.Description>
+        <Card.Description>
+          {this.nightView()}
+          <strong>{this.nightNumber(props.user)}
+          </strong>
+        </Card.Description>
+      </Card.Content>
+      <Card.Content >
+        <Grid.Column>
+          Hours Rank: {this.hourRank(props.user)}
+          {this.trophy(props.user, "hour")}
+        </Grid.Column>
+        <Grid.Column>
+          Nights Rank: {this.nightRank(props.user)}
+          {this.trophy(props.user, "night")}
+        </Grid.Column>
+      </Card.Content>
+    </Card>
+  }
+
+  //Needed to updated 'total' state value while switching view in dropdown. Look at line 192
+  handleChange(e, timeofDay) {
+    const total = timeofDay === 'Week'
+      ? false
+      : true;
+    this.setState({
+      total
+    }, () => {
+      this.handleNightorHours(e, this.state.rank)
+    });
+  };
+
+  // Trying to move the dropDown options out of the render component, but dropDown fucntion  is buggy
+
+  dropDown = () => {
```

Comment:
I would consider making this and the individual Cards' content left aligned. Just for consistency.